### ISSUE TITLE
[2021/08/12]推薦・機械学習勉強会

### ISSUE THREAD

**Issue by yu-ya4** [wantedly/machine-learning-round-table#111]
## Why

推薦・機械学習勉強会は、推薦や機械学習、その周辺技術を通じてサービスを改善することにモチベーションのある人達の集まりです。ニュースやブログから論文まで、気になったものについてお互い共有しましょう！

発信のため、ここは **public** にしてあります。外部からの参加をご希望の方は松村(https://twitter.com/yu__ya4) まで DM を送るか、Wantedly Visit の募集（https://www.wantedly.com/projects/391912） よりご連絡ください！

## What

Wantedly では隔週木曜日に

- 推薦の評価指標について議論したい
- 〇〇っていうライブラリ / フレームワークを導入してみたい
- 他社の基盤事例をみんなにシェアして自社の基盤開発に活かしたい
- もっと推薦を良くするためにどんなものが必要か議論したい

といった話をする「推薦・機械学習勉強会」を開催しています。
この ISSUE はその会で話すネタを共有するための場所です。

話したいことがある人はここにコメントしましょう！
会の間に話した内容もここにメモしましょう！

prev: #109

**Comment by hakubishin3** [comment]
## Nanka

**Comment by nogawanogawa** [comment]
# 何か書く(特に見当たらなければ[Fiber](https://eng.uber.com/fiberdistributed/))

**Comment by zerebom** [comment]
## [試行回数の増やし方 2021年度版](https://speakerdeck.com/butsugiri/increasing-number-of-attempts-ver-2021)
主に機械学習などの実験を行うに際して、如何に試行回数を増やして成功を得るかについてまとめてある記事。
例えば以下のような内容が書かれている。
- 実装編
    - 出戻りがなく小さく始められる実装順番
    - コードの分割
    - log・処理状況の可視化
- コード管理編
    - サーバーとローカルの自動同期方法
    - 実験の再現方法の確立
- 道具立て編
    - zshの整備(starship, fzf,etc.)

## [『機械学習を解釈する技術』＆『施策デザインのための機械学習入門』著者陣によるクロストーク 第1回](https://gihyo.jp/news/interview/01/ml_engineer/0001?page=1)
『機械学習を解釈する技術』＆『施策デザインのための機械学習入門』の著者である、安井さん、森下さん、斎藤さんの対談記事。
なぜ、これらの書籍を書くに至ったか、どういうモチベーションで書いたかについて語られている。
斎藤さんは自分にしか書けない書籍になるようにこだわったとのこと。自分もブログを書くときなどに意識したい。


## じっくり読めてないけど、興味のある記事
- [MNTSQ & UBIEでVERTICAL AI STARTUP MEETUPを開催しました](https://yag-ays.github.io/article/mntsq-ubie/)
- [カルマンフィルターが自動運転の自己位置推定で使われるまで](https://tech.tier4.jp/entry/2021/08/04/160000)
- [Pytorchの「.detach()」と「with no_grad():」と「.requires_grad = False」の違い](https://qiita.com/tttamaki/items/28f13a1507eb63387901) 
- [atmaCup#11 1st place solution](https://speakerdeck.com/ishikei/atmacup-number-11-1st-place-solution)

**Comment by yu-ya4** [comment]
## 「解釈性」と「説明性」は違う

https://twitter.com/shunk031/status/1424593975617327113

https://www.slideshare.net/YunyaoLi/explainability-for-natural-language-processing-249912819/29

けっこう同じ文脈で使っちゃってたかも反省。

<img width="810" alt="Explainability_for_Natural_Language_Processing" src="https://user-images.githubusercontent.com/12006023/129178494-a4e5db2d-fa4f-49ba-92fa-c19bac18c531.png">

今週インプット少なかったのも反省。。。
